tests/endpoints/cherrypick_test_data/test_valid_input.py

```diff
@@ -0,0 +1,140 @@
+import json
+import re
+from http import HTTPStatus
+from typing import List
+from unittest.mock import Mock, patch
+
+import pytest
+from requests.models import HTTPError, Response
+
+from lighthouse.constants.cherrypick_test_data import (
+    CPTD_STATUS_PENDING,
+    FIELD_CRAWLER_RUN_ID,
+)
+from lighthouse.constants.error_messages import ERROR_CRAWLER_HTTP_ERROR
+
+ENDPOINT_PATH = "/cherrypick-test-data"
+
+
+@pytest.fixture
+def mock_post():
+    with patch("requests.post") as mock_post:
+        yield mock_post
+
+
+def mock_response(status_code, json_object, http_error_msg=""):
```

Comment:
I think you might find https://github.com/getsentry/responses helpful, we use it somewhere else too.

Reply:
Why didn't I see this before?  I searched all over for something like this and didn't want to introduce any new dependencies to the project.  This is exactly what I needed and already here!

Reply:
Cool, I actually forgot it was already in the project too.